examples/deployment/docker/log_signer/Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM golang:1.9
 
-ENV DB_USER=test \
-    DB_PASSWORD=zaphod \
-    DB_DATABASE=test \
-    DB_HOST=127.0.0.0:3306
+ENV DB_FLAG="--mysql_url=test:zaphod@tcp(127.0.0.1:3306)/test" \
+    DB_PROVIDER="mysql"
 
 ENV HOST=0.0.0.0 \
     HTTP_PORT=8091
 
 ENV SEQUENCER_GUARD_WINDOW=0s \
     FORCE_MASTER=true \
-    SEQUENCER_INTERVAL=10s \
+    SEQUENCER_INTERVAL=300ms \
```

Comment:
Seems like a big reduction?

Reply:
The signer needs to run at least once a second, right?
Otherwise, if you imagine a signing cadence which is a multiple of the number of buckets then there's a risk that things will take a very long time to get sequenced...

Reply:
True for CloudSpanner but possibly not for MySQL.